From 7e3c22174a49f03046f4004e5acad7665a7c86b2 Mon Sep 17 00:00:00 2001
From: Hugo Posnic <hugo.posnic@gmail.com>
Date: Fri, 13 Sep 2019 18:02:48 +0200
Subject: [PATCH 1/1] Finally fix #20

---
 src/main.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/main.py b/src/main.py
index 93a24c4..abd640b 100755
--- a/src/main.py
+++ b/src/main.py
@@ -30,11 +30,10 @@ class Application(Gtk.Application):
         self.win.present()
 
     def file_open_handler(self, app, g_file_list, amount, ukwn):
+        self.do_activate()
         for g_file in g_file_list:
             self.win.open_image(filename=g_file.get_path())
 
-        self.do_activate()
-
 
 def main(version):
     app = Application()
-- 
2.21.0

